### ISSUE TITLE
ImportError: /usr/lib/libgdal.so.1: undefined symbol: nc_def_var_deflate

### ISSUE THREAD

**Issue by YudanJIN** [IaPCS/gmsh-exodus-converter#4]
I have installed libvtk6.2 and python-vtk6. When I run 
`python /home/yudan/Downloads/gmsh-exodus-converter-master/convert.py -i /home/yudan/Desktop/test.msh -o test.g -t 4`
I have:
> 
Traceback (most recent call last):
  File "/home/yudan/Downloads/gmsh-exodus-converter-master/convert.py", line 10, in <module>
    from vtk import vtkExodusIIWriter
  File "/usr/lib/python2.7/dist-packages/vtk/__init__.py", line 113, in <module>
    from vtkIOGDAL import *
  File "/usr/lib/python2.7/dist-packages/vtk/vtkIOGDAL.py", line 1, in <module>
    from vtkIOGDALPython import *
ImportError: /usr/lib/libgdal.so.1: undefined symbol: nc_def_var_deflate

**Comment by diehlpk** [comment]
Could you may provide your test.msh so we can try to reproduce the error?

**Comment by diehlpk** [comment]
And the error message indicates, that there is a problem with your installation of vtk
```from vtk import vtkExodusIIWriter
File "/usr/lib/python2.7/dist-packages/vtk/init.py", line 113, in
from vtkIOGDAL import *
File "/usr/lib/python2.7/dist-packages/vtk/vtkIOGDAL.py", line 1, in
from vtkIOGDALPython import *
ImportError: /usr/lib/libgdal.so.1: undefined symbol: nc_def_var_deflate```

**Comment by YudanJIN** [comment]
Thanks for your quick answer, I reinstall the python-vtk by `apt install python-vtk`, but the version is 5.10.1. Now it worked.
![screenshot from 2017-03-21 17-24-37](https://cloud.githubusercontent.com/assets/26431272/24157985/4b1e12dc-0e5b-11e7-8135-b0ecfce0d3bf.png)

**Comment by ilyasst** [comment]
Glad to hear that !
Just out of curiosity (we'd like to know what purpose the code serves) @YudanJIN , could you let us know what do you need an Exodus file for ? 

Thanks !

**Comment by YudanJIN** [comment]
To change the file .msh to .g to be used in Peridigm. I just started to use peridigm, there are lots of things I don't know. 
I remplaced the discretisation "Text File" in the exemples of Peridigm: fragmenting_cylinder.peridigm
After run, I have:
![screenshot from 2017-03-21 17-51-04](https://cloud.githubusercontent.com/assets/26431272/24159278/01c86354-0e5f-11e7-9e36-71a020d16998.png)
Is that because I can't just run it by only remplacing the text file.
